plugins/inputs/win_eventlog/win_eventlog.go

```diff
@@ -113,7 +113,7 @@ func (w *WinEventLog) Stop() {
 }
 
 func (w *WinEventLog) GetState() interface{} {
-	bookmarkXML, err := w.renderBookmark(w.bookmark)
+	bookmarkXML, err := renderBookmark(w.bookmark)
```

Comment:
What about
```suggestion
	bookmarkXML, err := w.renderBookmark()
```

Reply:
Many methods in this plugin might be unexported. I will be fixing this as part of the work on the `revive:exported` linter.

Reply:
Your proposal is one of the correct solutions, just like mine. 😎